languages/elixir/exercises/concept/agent/lib/community_garden.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Plot do
+  raise "Please implement the Plot struct"
```

Comment:
What would you think about providing the implementation in the boilerplate and removing the first step of the exercise? There are 6 steps in total right now (that's quite a lot), and this exercise is about using `Agent`, structs were already taught.

Reply:
Sure

Reply:
Goo idea